Source/ACE/Entity/Container.cs

```diff
@@ -26,48 +29,111 @@ public Container(ObjectType type, ObjectGuid guid, string name, ushort weenieCla
         public Container(AceObject baseObject)
             : base(baseObject) { }
 
+        public Container(ObjectGuid guid, AceObject baseAceObject)
```

Comment:
I probably got confused.   It happens :)     That debug object - I don't understand why we have it.   I am not saying it is wrong.   It can't be used for containers like backpacks and it was causing me some real grief.    I will fix it.   Might be later this afternoon before I can work on it.